llmfoundry/models/mpt/configuration_mpt.py

```diff
@@ -222,6 +223,20 @@ def _validate_config(self) -> None:
                 'attn_impl'] not in ['torch', 'triton']:
             raise NotImplementedError(
                 'prefix_lm only implemented with torch and triton attention.')
+
+        if self.attn_config['attn_impl'] == 'flash' and is_flash_v1_installed():
```

Comment:
add the instructions for how to install and what will change in future version. so, say, you can install with .[gpu-flash2], and after deprecation .[gpu] will be updated to install flash2

Reply:
Added the instruction on how to install Flash 2 in the warning, and updated readme to say that after we remove support for FA v1, .[gpu] will be updated to install flash2.